packages/server-core-integration/README.md

```diff
@@ -72,7 +72,7 @@ ddp.connect((err: any) => {
 		return
 	}
 	let subId = ddp.subscribe('expectedMediaItems')
```

Comment:
this should be a PubSub reference, not a string, right?

Reply:
Actually no. This is the `DDPClient`, describing how to use the connection class independently from Sofie. 
Instead I've changed the line below back to not be using an enum